src/web/browser/ga/ga.ts

```diff
@@ -212,3 +212,24 @@ export const trackPerformance = (
         );
     }
 };
+
+export const videoTracking = ({
```

Comment:
Can re rename this to something more imperative? A name that is more of a verb really helps my brain figure out what thins actually _do_

Reply:
Ideally, the naming would also follow the convention already established in the file. Maybe `trackVideos`? Or `trackVideoInteraction`?